stata_kernel/stata_session.py

```diff
@@ -58,6 +58,7 @@
 class StataSession(object):
     def __init__(self):
 
+        self.install = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Can we rename `self.install` to something like `self.adodir`? And I'd prefer
```
Path(__file__).resolve().parent
```

Reply:
Sure.

Reply:
Actually sorry, this is to give to Stata to run the adofile? It seems safer(?) to use `pkg_resources.resource_filename`. Maybe safer across different methods of installing the package?

https://stackoverflow.com/a/39659

```py
import pkg_resources as pkg
pkg.resource_filename('stata_kernel', 'ado/_StataKernelCompletions.ado')
```